AdminWebsite/AdminWebsite/Contracts/Responses/PublicHolidayResponse.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace AdminWebsite.Contracts.Responses
+{
+    /// <summary>
+    /// A public holiday
+    /// </summary>
+    public class PublicHolidayResponse
+    {
+        /// <summary>
+        /// Name of a public holiday
+        /// </summary>
+        public string Name { get; set; }
```

Comment:
some holidays are specific to some regions, we support welsh now, just checking this has been discussed at some point

Reply:
The name gets pulled in from the GOVUK API, I pull in the names they provide which include England and Wales